cluster/gce/configure-vm.sh

```diff
@@ -968,7 +968,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: 10.123.45.0/30
+  cbr-cidr: 10.123.45.0/29
```

Comment:
for now the cidr is not real. actually i think the master node is assigned a /24 because the node controller doesn't know any better (and this is real, it has a route), but we just use this hardcoded one. 

I was worried about a case like proxy/exec/logs where the master might use the internal ip of some node, and if that collides with fluents ip because of operation error (not an issue on platforms that use external ip like gke).  

https://github.com/kubernetes/kubernetes/issues/32900